### DISCUSSION TITLE
Finding benchmark dataset - node prediction + binary classes + high number of graphs

### DISCUSSION THREAD

**Discussion by kimzed** [pyg-team/pytorch_geometric#4618] [General]
Hello everybody,

For my research I would like to test my models and my training pipeline with a benchmark dataset with similar characteristics as my own dataset to be sure that the model architecture / training pipeline has no error bug. I looked into various torch_geometric.datasets but it is difficult to match what i need.

Does anybody knows about a benchmark dataset from torch geometric that matches these criterias?

 node prediction, binary classes, high number of graphs

**Comment by rusty1s** [comment]
I am not aware of any datasets many datasets with these characteristics. The closest would probably be `PPI` dataset (but it is multi-label prediction on only 24 graphs), and the `PATTERN` and `CLUSTER` datasets from the `GNNBenchmarkDataset` (but they are multi-class classification).

**Comment by kimzed** [comment]
PATTERN really looks like the dataset that I need. thanks a lot, i used it and it works.
